### PR TITLE
Corrected milestone bug and using keyVault for passwords

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

**Comment by Anipik** [comment]
yes this tool was removing the milestone from various issue in corefx repos. so we have to take it offline.

File: src/Microsoft.DotNet.Github.IssueLabeler/Models/Labeler.cs [hunk 1]
```diff
@@ -42,6 +75,7 @@ public async Task PredictAndApplyLabelAsync(int number, string title, string bod
             {
                 var issueUpdate = new IssueUpdate();
                 issueUpdate.AddLabel(label);
+                issueUpdate.Milestone = 1;
```
Comment (danmosemsft) [inline src/Microsoft.DotNet.Github.IssueLabeler/Models/Labeler.cs]: How does this fix the issue of resetting the milestone? Shouldn't it reapply the milestone the issue already had (if any)?

Comment (Anipik) [reply src/Microsoft.DotNet.Github.IssueLabeler/Models/Labeler.cs]: It tells github the issue the maximum number of milestone that this issue can have.
By default the value is null which implies it cant have a milestone hence is removed

Comment (Anipik) [reply src/Microsoft.DotNet.Github.IssueLabeler/Models/Labeler.cs]: eg https://github.com/Anipik/Compiler/issues/18

Comment (danmosemsft) [reply src/Microsoft.DotNet.Github.IssueLabeler/Models/Labeler.cs]: What a poor choice for their API name then, it should be "MilestoneCount" presumably. Worth a comment in the code.

**Pull request merged**